src/main/java/org/truffleruby/core/rope/RopeOperations.java

```diff
@@ -127,6 +127,22 @@ public static String decodeUTF8(byte[] bytes, int byteOffset, int byteLength) {
         return decode(UTF8, bytes, byteOffset, byteLength);
     }
 
+    public static Rope encodeAscii(CharSequence value, Encoding encoding) {
```

Comment:
Is it useful to type this as `CharSequence` and not just `String`?
I'd rather be specific unless there is a need.

Reply:
I'll take a look. I matched the signature for `StringOperations.encodeRope`.